.github/workflows/centos_8.yml

```diff
@@ -36,9 +35,11 @@ concurrency:
 
 jobs:
   centos_8:
-    # Run on pull request only if the 'full-ci' label is set.
-    if: github.event_name != 'pull_request' ||
-        contains(github.event.pull_request.labels.*.name, 'full-ci')
+    # Run on push to the 'master' and release branches of tarantool/tarantool
+    # or on pull request if the 'full-ci' label is set.
+    if: ( github.event_name != 'pull_request' &&
+      github.repository == 'tarantool/tarantool' ) ||
+      contains(github.event.pull_request.labels.*.name, 'full-ci')
```

Comment:
Technically, it's possible to create a PR for a fork so I guess the condition should look like this:

```yaml
    if: github.repository == 'tarantool/tarantool' ) &&
        ( github.event_name != 'pull_request' ||
          contains(github.event.pull_request.labels.*.name, 'full-ci') )
```